src/splitInteger.test.js

```diff
@@ -2,20 +2,20 @@
 
 const splitInteger = require('./splitInteger');
 
-test(`should split a number into equal parts
+it(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(2, 2)).toEqual([1, 1]);
 });
 
-test(`should return a part equals to a value
+it(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(17, 4)).toHaveLength(4);
 });
 
-test('should sort parts ascending if they are not equal', () => {
-
+it('should sort parts ascending if they are not equal', () => {
+  expect(splitInteger(32, 6)).toEqual([5, 5, 5, 5, 6, 6]);
 });
 
-test('should add zeros if value < numberOfParts', () => {
-
+it('should add zeros if value < numberOfParts', () => {
```

Comment:
Passing 0 as the number of parts (numberOfParts) seems to be an error. Typically, the number of parts should be a positive integer greater than zero. Dividing a number into zero parts doesn't make sense mathematically, and it's likely to cause issues in the splitInteger function, potentially leading to unexpected behavior or errors.
Choose a valid number of parts greater than zero but smaller than 17 for testing this scenario.